Sources/GRPCProtobufCodeGen/ProtobufCodeGenParser.swift

```diff
@@ -23,7 +23,14 @@ import struct GRPCCodeGen.CodeGenerationRequest
 /// Parses a ``FileDescriptor`` object into a ``CodeGenerationRequest`` object.
 internal struct ProtobufCodeGenParser {
   internal init() {}
-  internal func parse(input: FileDescriptor) throws -> CodeGenerationRequest {
+  internal func parse(
+    input: FileDescriptor,
+    protoFileToModuleMappings: ProtoFileToModuleMappings
```

Comment:
Do we actually need the mappings? It looks like we only actually need the namer. Also I think it makes more sense to configure the parser with the namer/mappings rather than passing it per call to `parse`.